src/main/java/com/edmunds/tools/databricks/maven/UpsertClusterMojo.java

```diff
@@ -54,23 +51,24 @@
  * Cluster mojo, to perform databricks cluster upsert (create or update through recreation).
  */
 @Mojo(name = "upsert-cluster", requiresProject = true)
-public class UpsertClusterMojo extends BaseDatabricksMojo {
-
-    /**
-     * The databricks cluster json file that contains all of the information for how to create databricks cluster.
-     */
-    @Parameter(defaultValue = "${project.build.resources[0].directory}/databricks-plugin/databricks-cluster-settings.json", property = "dbClusterFile")
-    protected File dbClusterFile;
+public class UpsertClusterMojo extends BaseDatabricksUpsertClusterMojo {
 
+    @Override
     public void execute() throws MojoExecutionException {
-        ClusterTemplateModel[] cts = getClusterTemplateModels();
-        if (cts.length == 0) {
+        upsertJobSettings();
+    }
+
+    private void upsertJobSettings() throws MojoExecutionException {
+        List<ClusterTemplateDTO> cts = getSettingsUtils().buildTemplateDTOsWithDefaults();
+        if (cts.size() == 0) {
             return;
```

Comment:
does it log elsewhere?

Reply:
I've added a logging point for that case